### DISCUSSION TITLE
http handling. choosing a third party library

### DISCUSSION THREAD

**Discussion by maxwww** [abdukhashimov/student_aggregator#24] [Ideas]
Hi team
As our App should handle HTTP requests we are going to use some third-party libraries to simplify and speed up feature development.

Madiyor suggested 3 libraries:
gin
go fiber
gorilla mux

I did a little comparing these libs and it looks like any of the libs is a great candidate for meeting our needs.
They have similar powerful functionality and it is very easy to use any of them and they are very popular on Github
But my choice is gorilla/mux.
And my choice is based only on that gorilla/mux is just a router (gin and fiber are complete web frameworks) and we will learn Go more deeply in this way.

@abdukhashimov @davit312 @ADrema @abdukhashimov @arman42117 @ihoro-epam Please provide your thoughts about this.

**Comment by abdukhashimov** [comment]
I also support the gorilla/mux, in case we need to use grpc-gateway, we can do some modification there as well. It also uses gorilla/mux

**Comment by ihoro-epam** [comment]
+1 to gorila/mux.
